docs/sources/alerting/alerting-rules/manage-contact-points/integrations/_index.md

```diff
@@ -2,7 +2,7 @@
 aliases:
   - alerting/manage-notifications/manage-contact-points/configure-integrations/
 canonical: https://grafana.com/docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/
-description: Configure integrations
+description: Configure contact point integrations to select your preferred communication channels for receiving notifications of firing alerts.
```

Comment:
Should all descriptions end in a full stop?

Reply:
I don't know! All of mine had none, so i just left them out - but it's something i hadn't thought of that we should decide on as a team